wordpress-mobile/WordPress-Android
# Deprecates old in-line notification icons

Fixes #20022

## Description
This PR removes the notification avatar icons along with the related code and resources

|Before|After|
|---|---|
|![Screenshot_20240129_190927](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/d9d11acf-78eb-4957-897a-88490dbcc0df)|![Screenshot_20240129_190915](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/4f10078e-fabe-4908-b9f0-67c53861f228)|

-----

## To Test:
1. Open the Jetpack app and switch to the notifications tab
2. *Verify* that the badges are removed from the notifications
3. *Verify* that the functionality remains unaffected



-----

## Regression Notes

1. Potential unintended areas of impact

    - Notifications tab

4. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

5. What automated tests I added (or what prevented me from doing so)

    - Removed the test cases related to the changed code.

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
